src/Server/RestRouterHandlers/RawstoreTableRestRouterHandler.cpp

```diff
@@ -52,6 +52,32 @@ void RawstoreTableRestRouterHandler::buildTablesJSON(Poco::JSON::Object & resp,
         table_mapping_json.set("order_by_expression", table->sorting_key);
         table_mapping_json.set("partition_by_expression", table->partition_key);
 
+        const auto & catalog_service = CatalogService::instance(query_context);
```

Comment:
Could we reuse the code block as they are identical.

Reply:
Yes